OutOfSchool/OutOfSchool.WebApi/Services/Database/IWorkshopService.cs

```diff
@@ -41,5 +41,7 @@ public interface IWorkshopService : ICRUDService<WorkshopDTO, Guid>
         /// <returns>A <see cref="Task{TResult}"/> representing the result of the asynchronous operation.
         /// The task result contains the <see cref="SearchResult{WorkshopCard}"/> that contains found elements.</returns>
         Task<SearchResult<WorkshopCard>> GetNearestByFilter(WorkshopFilter filter = null);
+
+        Task<IEnumerable<WorkshopDTO>> GetByIds(IEnumerable<Guid> ids);
```

Comment:
do we really need to return DTO? see `ElasticsearchSynchronizationService.cs` comment